src/pepr/operator/crd/generated/clusterconfig-v1alpha1.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright 2025 Defense Unicorns
+ * SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+ */
+
+// This file is auto-generated by kubernetes-fluent-client, do not edit manually
+import { GenericKind, RegisterKind } from "kubernetes-fluent-client";
+export class ClusterConfig extends GenericKind {
+  spec?: Spec;
+}
+
+export interface Spec {
+  attributes?: Attributes;
+  expose: Expose;
+  networking?: Networking;
+  policy: Policy;
+}
+
+export interface Attributes {
+  /**
+   * Friendly name to associate with your UDS cluster
+   */
+  clusterName?: string;
+  /**
+   * Tags to apply to your UDS cluster
+   */
+  tags?: string[];
+}
+
+export interface Expose {
+  /**
+   * Domain all cluster services on the admin gateway will be exposed on
+   */
+  adminDomain?: string;
+  /**
+   * The trusted CA that signed your domain certificates if using Private PKI
+   */
+  caCert?: string;
+  /**
+   * Domain all cluster services will be exposed on
+   */
+  domain: string;
+}
+
+export interface Networking {
+  /**
+   * cidr range for your Kubernetes control plane nodes. This is a manual override that can be
+   * used instead of relying on Pepr to automatically watch and update the values
+   */
+  kubeapiCIDR?: string;
+  /**
+   * cidr(s) for all Kubernetes nodes (not just control plane). Similar reason to above,annual
+   * override instead of relying on watch
+   */
+  kubenodeCIDRS?: string[];
```

Comment:
Do we care about configuring other CIDRs like Services?

Reply:
I think for now this is just to keep track of the 2 cidrs / cidr lists that we need for various network policies (in particular for kubeapi and kube node access). In theory could add kube services / kube pods as well but might defer that to when we have a usecase for tracking/using those values?